README.md

```diff
@@ -78,6 +80,7 @@ var target = appResolver.getUrl();
   - `dist` - The directory containing the app files to serve.  By default, the `dist` directory is used.
   - `port` - The port to listen on.  By default, port `3000` is used, which is the port that the LMS expects it on.
   - `hostname` - The hostname (or IP) to listen on. By default, the hostname of the operating system is used.  You should not need to change this.
+  - `publicEndpoint` - If provided overrides the protocol (http) hostname and port for endpoint resolution
```

Comment:
Another option would be to make the new parameter `protocol`, so then you could use the existing `port` and `hostname` options and just set `protocol` = `https`. 🤷

Reply:
I tried that first, but I ran into trouble when trying to run the server. I only want to override the value which gets written into the `appconfig.json` file, i.e. for the loader endpoint.

Reply:
Cool, makes sense!